sklearn/model_selection/_search.py

```diff
@@ -454,6 +473,14 @@ def score(self, X, y=None):
         """
         _check_refit(self, "score")
         check_is_fitted(self)
+
+        _raise_for_params(params, self, "score")
```

Comment:
I think we might be missing a test for when an exception should be raised?

Reply:
Done